pkg/resource/cluster/hooks.go

```diff
@@ -102,3 +108,160 @@ func clusterDeleting(r *resource) bool {
 	cs := *r.ko.Status.State
 	return cs == svcsdk.ClusterStateDeleting
 }
+
+func (rm *resourceManager) customUpdate(
+	ctx context.Context,
+	desired *resource,
+	latest *resource,
+	delta *ackcompare.Delta,
+) (updated *resource, err error) {
+	rlog := ackrtlog.FromContext(ctx)
+	exit := rlog.Trace("rm.customUpdate")
+	defer exit(err)
```

Comment:
non-blocking: i know this is something used all over the place in ack, we recently learned that the correct defer is to:
```suggestion
	defer func() { exit(err) }()
```